src/FplBot.Core/Helpers/Formatting/Formatter.cs

```diff
@@ -293,6 +293,22 @@ public static string FormatPriceChanged(IEnumerable<Player> priceChangesPlayers,
             return messageToSend;
         }
 
+        public static string FormatNewPlayers(IEnumerable<NewPlayer> newPlayers)
+        {
+            var header = $"🆕 New player!";
+            if (newPlayers.Count() > 1)
+            {
+                header = "🆕 New players!";
+            }
+            var nameAndCost = newPlayers.Select(NameAndCost);
+            return $"{header}\n{BulletPoints(nameAndCost)}";
+
+            string NameAndCost(NewPlayer p)
+            {
+                return $"{p.Player.WebName}({p.Team.ShortName}) {FormatCurrency(p.Player.NowCost)}";
```

Comment:
Missing space between player’s name and team’s name

Reply:
```suggestion
                return $"{p.Player.WebName} ({p.Team.ShortName}) {FormatCurrency(p.Player.NowCost)}";
```